packages/@aws-cdk/aws-servicecatalog/test/portfolio.test.ts

```diff
@@ -0,0 +1,271 @@
+import '@aws-cdk/assert-internal/jest';
+import { AccountRootPrincipal, Group, Role, User } from '@aws-cdk/aws-iam';
+import { App, Stack, Tags } from '@aws-cdk/core';
+import { AcceptLanguage, Portfolio } from '../lib';
+
+describe('Portfolio', () => {
+
+  test('default portfolio creation', () => {
+    const app = new App();
+    const stack = new Stack(app);
```

Comment:
These two lines are copy&pasted basically in every test. Can you extract them to a `beforeEach()` instead?

(BTW, pretty sure there's no need for `app` here)

Reply:
yeah the `app` part is from larger request where we needed unique names for anything with an association to add to our hash map.  But I think we will follow recommendation and have these simpler tests in their own block that can do this before each and keep ones that need unique name in different block

Reply:
Oh, if you do indeed need `App` in those tests, that's fine - by all means, keep it as-is. It's just rare for these kinds of unit tests to need it, so you can usually save a bit of the setup. But if your case requires it, that's totally fine.